ppml/README.md

```diff
@@ -233,9 +231,69 @@ To build your own Big Data & AI applications, refer to [develop your own Big Dat
 
 #### Step 4. Attestation 
 
-1. Disable attestation
+1. Start BigDL PPML Client Container
+   
+   First, create K8S RBAC:
+   
+   ```
+   kubectl create serviceaccount spark
+   kubectl create clusterrolebinding spark-role --clusterrole=edit --serviceaccount=default:spark --namespace=default
+   kubectl get secret|grep service-account-token # you will find a spark service account secret, format like spark-token-12345
+
+   # bind service account and user
+   kubectl config set-credentials spark-user \
+                  --token=$(kubectl get secret <spark_service_account_secret> -o jsonpath={.data.token} | base64 -d)
+ 
+   # bind user and context
+   kubectl config set-context spark-context --user=spark-user
+   
+   # bind context and cluster
+   kubectl config get-clusters
+   kubectl config set-context spark-context --cluster=<cluster_name> --user=spark-user
+   ```
+   
+   Second, create kubeconfig and upload it as k8s secret for k8s authentication:
+   
+   ```
+   kubectl config use-context spark-context
+   kubectl config view --flatten --minify > /YOUR_DIR/kubeconfig
+   ```
 
-    To disable attestation service, you should configure spark-driver-template.yaml and spark-executor-template.yaml to set `ATTESTATION` value to `false`. By default, the attestation service is disabled. 
+   Third, start a client container:
+   
+   ```
+   export K8S_MASTER=k8s://$(sudo kubectl cluster-info | grep 'https.*6443' -o -m 1)
+   echo The k8s master is $K8S_MASTER .
+   export DATA_PATH=/YOUR_DIR/data
+   export KEYS_PATH=/YOUR_DIR/keys
+   export SECURE_PASSWORD_PATH=/YOUR_DIR/password
+   export KUBECONFIG_PATH=/YOUR_DIR/kubeconfig
+   export LOCAL_IP=$LOCAL_IP
+   export DOCKER_IMAGE=intelanalytics/bigdl-ppml-trusted-big-data-ml-python-gramine-reference:2.2.0-SNAPSHOT # or the custom image built by yourself
+    
+   sudo docker run -itd \
+       --privileged \
+       --net=host \
+       --name=bigdl-ppml-client-k8s \
+       --cpuset-cpus="0-4" \
+       --oom-kill-disable \
+       --device=/dev/sgx/enclave \
+       --device=/dev/sgx/provision \
+       -v /var/run/aesmd/aesm.socket:/var/run/aesmd/aesm.socket \
+       -v $DATA_PATH:/ppml/trusted-big-data-ml/work/data \
+       -v $KEYS_PATH:/ppml/trusted-big-data-ml/work/keys \
+       -v $SECURE_PASSWORD_PATH:/ppml/trusted-big-data-ml/work/password \
+       -v $KUBECONFIG_PATH:/root/.kube/config \
+       -e RUNTIME_SPARK_MASTER=$K8S_MASTER \
+       -e RUNTIME_K8S_SPARK_IMAGE=$DOCKER_IMAGE \
+       -e LOCAL_IP=$LOCAL_IP \
+       $DOCKER_IMAGE bash
+    ``` 
```

Comment:
It seems line234 - line291 could be put into prepare_environment.md as it has the same content.

Reply:
Maybe we should delete start-client-container section from prepare_environment.md. The prepare_environment is step 0 while the client container needs to be started after customer image building.

Reply:
What about put start-client-container section in the end of **Step 1. Build your PPML image for production environment** (after create custom image) and update K8S RBAC section in prepare_environment.md?

Reply:
That's great.